e2e/testcases/apiservice.bats

```diff
@@ -13,17 +13,32 @@ YAML_DIR=${BATS_TEST_DIRNAME}/../testdata/apiservice
 test_setup() {
   setup::git::initialize
   setup::git::commit_minimal_repo_contents
+  wait::for -t 120 -- nomos::repo_synced
 }
 
 test_teardown() {
   git::rm acme/cluster/apiservice.yaml
   git::commit
-  kubectl delete -f "${YAML_DIR}/apiservice.yaml" --ignore-not-found
+  wait::for -t 120 -- nomos::repo_synced
+kubectl delete -f "${YAML_DIR}/apiservice.yaml" --ignore-not-found
+  git::rm acme/namespaces/custom-metrics/namespace.yaml
+  git::rm acme/namespaces/custom-metrics/namespace-custom-metrics.yaml
+  git::commit
+  kubectl delete -f "${YAML_DIR}/namespace.yaml" --ignore-not-found
+  kubectl delete -f "${YAML_DIR}/namespace-custom-metrics.yaml" --ignore-not-found
+  kubectl delete -f "${YAML_DIR}/rbac.yaml" --ignore-not-found
+
   setup::git::remove_all acme
 }
 
 @test "Create APIService and endpoint in same commit" {
+  kubectl apply -f "${YAML_DIR}/namespace.yaml"
+  kubectl apply -f "${YAML_DIR}/rbac.yaml"
```

Comment:
Why do you apply them manually using kubectl? This testcase is supposed to test the resources being managed by Config Sync.

Reply:
Moved into commit by Config Sync. To not having to breakdown the test data and create more artifacts I'm using unstructured source format for this test bats.